_posts/2021-08-25-nlog-5-0-preview1-ready.md

```diff
@@ -187,6 +187,31 @@ Having the logging-layer making calls into the dependency-injection-layer can ca
 or deadlocks. Both issues are extremely annoying to debug and diagnose, so one should be very careful to ensure interfaces
 are implemented with singleton-lifetime.
 
+### Aliases support for targets, layouts, layout renderers and conditions
+
+The following attributes are now allowed multiple times for a class:
+
+- `[Target]`
+- `[Layout]`
+- `LayoutRenderer]`
+- `[Filter]`
+
+This will give the possibilty to add aliases for your targets, layouts, layout renderers and conditions.
+
+The following aliases are added: 
+- The Mail Target has now the aliases Email, SMTP and SMTP-Client
+- The Trace Target has now the alias TraceSystem
+- ${Event-properties} has now the alias {Event-property}
+- ${logger} has now the alias ${loggername}
+
+All aliases are also listed and searchable on https://nlog-project.org/config/
+
+### Dashes (-) will be ignored when parsing names
+Dashes in names in targets, layout renderers, layouts, filters are ignored. For example: `${loggername}` could be written als `${logger-name}`, and ColoredConsole could be written als Colored-Console.
```

Comment:
Maybe explain that it is the symbol-name / type-name that now has more relaxed handling, when using in NLog Configuration files.